plugins/modules/mso_schema_template_filter_entry.py

```diff
@@ -284,8 +292,10 @@ def main():
         if not mso.existing:
             if display_name is None:
                 display_name = entry
-            if description is None:
-                description = ''
+            if filter_description is None:
+                filter_description = ''
+            if filter_entry_description is None:
+                filter_entry_description = ''
```

Comment:
Shouldn't we use default for that? Why are we setting some modules description always to '' and other we don't?